docs/_docs/collections.md

```diff
@@ -55,10 +55,10 @@ defaults:
   in <code>my_collections/_recipes</code> for the <code>recipes</code> collection.</p>
 </div>
 
-<div class="note warning">
-  <h5>Be sure to move posts into custom collections directory</h5>
+<div class="note">
+  <h5>Moving posts into the custom collections directory</h5>
 
-  <p>If you specify a directory to store all your collections in the same place with <code>collections_dir: my_collections</code>, then you will need to move your <code>_posts</code> directory to <code>my_collections/_posts</code>.</p>
+  <p>If you want to move your `_posts` folder inside the custom collections directory as well, the name of the `collections_dir` cannot start with an underscore (_).</p>
```

Comment:
There is no scope for a *"If you want to move `_posts`.."*
Its always, "If you specify a custom directory.., then `_posts` **has to be moved** into it as well.."
and since `_posts` won't be read inside directories starting with an underscore, it implies:
```
Custom collections_dir cannot start with an underscore
```

Reply:
Hmmm, that’s not the behaviour I’m seeing. I can set a custom collections directory, **and** have my `_posts` folder outside of it. The problem only occurs if I move `_posts` into a the `collections_dir` (and it begins with an underscore).

Reply:
Please check the generated URL to your posts..
`./_posts/2018-01-18-hello.md` will have generated a `./_site/_posts/2018/01/18/hello.html` .. (Note the `_posts` in your URL..)

Reply:
Ah yes, I hadn’t spotted that in my test site (and for my own purposes, I am using front matter defaults to explicitly set permalink for posts). Okay, so how about this:

> If you specify a directory to store all your collections in the same place with `collections_dir: my_collections`, then you will need to move your `_posts` directory to `my_collections/_posts`. Note that if you do this, the name of your collections directory cannot start with an underscore (_).

Reply:
Thanks @paulrobertlloyd, 👍 for your last proposal

Reply:
Great. Do you need me to update this PR?

Reply:
IMO, there's no need for two if statements..
> **If you specify a directory to store all your collections in the same place with `collections_dir: my_collections`, then you will need to move your` _posts` directory to `my_collections/_posts`. Note that, the name of your collections directory cannot start with an underscore (`_`).**


> Great. Do you need me to update this PR?

Yes. Please..

Reply:
@ashmaroli I wasn’t sure about that, but then realised we are adding this note within the context of moving `_posts` into the custom directory. So yes, no need for the extra ‘if’. I have updated this PR.